### DISCUSSION TITLE
读叔本华《人生的智慧》 — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#256] [Announcements]
# 读叔本华《人生的智慧》 — 码志

最近读完了这本小书，想写点什么，但又似乎暂时表达不出，先挖个读书笔记的坑在这里，后面有所感了也许会填上。

https://mazhuang.org/2021/10/25/the-wisdom-of-life/

**Comment by wqw547243068** [comment]
- 赞
